src/SFA.DAS.ApplyService.Web/Views/Roatp/ApplicationRejected.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@using SFA.DAS.AdminService.Common.Extensions;
+@model SFA.DAS.ApplyService.Web.ViewModels.Roatp.ApplicationSummaryViewModel
+
+@{
+    ViewBag.Title = "Application rejected";
+    Layout = "_Layout";
+}
+
+<nav>
+    <ul id="navigation" class="roatp__navigation " aria-label="Top Level Navigation">
+        <li class="roatp__navigation-item">
+            <a class="govuk-link govuk-link--no-visited-state roatp__navigation-link--current">Application overview</a>
+        </li>
+        <li class="roatp__navigation-item roatp__navigation-item--signout">
+            <a class="govuk-link govuk-link--no-visited-state" asp-action="SignOut" asp-controller="Users">Save and sign out</a>
+        </li>
+    </ul>
+</nav>
+
+<main class="govuk-main-wrapper " id="main-content" role="main">
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <div class="govuk-panel app-panel--warning">
```

Comment:
It's probably a no but... wouldn't it been better to have this as _govuk-panel--warning_ or make the ones that use _govuk-panel--confirmation_ have a named one like _app-panel--confirmation_ so at least all styles are consistently named for the panels